theano/configdefaults.py

```diff
@@ -1246,6 +1246,11 @@ def check_mkl_openmp():
     if ('MKL_THREADING_LAYER' in os.environ and
             os.environ['MKL_THREADING_LAYER'] == 'GNU'):
         return
+    try:
+        import numpy._mklinit  # noqa
+        return
+    except ImportError:
+        pass
     try:
         import mkl
         if '2018' in mkl.get_version_string():
```

Comment:
I would update the error message below to mention updating the numpy conda package as the first option, rather than only setting MLK_THREADING, since it would fix the problem as well.

Reply:
done